.env.sample

```diff
@@ -0,0 +1,2 @@
+GITHUB_TOKEN=
```

Comment:
You may want to add comments to explain the purpose of these variables, especially if the code is shared with others or revisited in the future.